test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -550,23 +550,6 @@ define([
                         },
                     ],
                 },
-                {
```

Comment:
remove these tests for single messages and just use the tests for multiple messages